code/modules/hivemind/machines.dm

```diff
@@ -0,0 +1,486 @@
+//Hivemind various machines
+
+/obj/structure/hivemind_machine
```

Comment:
Why are these not /obj/machinery?

They should be subclasses of machine unless there's a specific reason not to